client/store/index.js

```diff
@@ -0,0 +1,27 @@
+import { createStore, applyMiddleware } from 'redux';
+import axios from 'axios';
+import appReducer from './redux';
+import { createLogger } from 'redux-logger';
+import { composeWithDevTools } from 'redux-devtools-extension';
+import thunkMiddleware from 'redux-thunk';
+
+let middleware = [
+  createLogger({ collapsed: true }),
+  thunkMiddleware.withExtraArgument({ axios }),
+];
+
+const RESET_STORE = 'RESET_STORE';
+
+export const resetStore = () => ({ type: RESET_STORE });
+const rootReducer = (state, action) => {
+  if (action.type === RESET_STORE) {
+    state = undefined;
```

Comment:
Is there a reason this exists? I'm worried it might be dangerous setting state to undefined

Reply:
lol. Fancy code out of nowhere. I myself wonder where this was referenced from.